lib/cadet_web/controllers/user_controller.ex

```diff
@@ -22,20 +24,75 @@ defmodule CadetWeb.UserController do
       grade: grade,
       max_grade: max_grade,
       story: story,
-      xp: xp
+      xp: xp,
+      game_states: game_states
     )
   end
 
   swagger_path :index do
     get("/user")
-
     summary("Get the name and role of a user")
-
     security([%{JWT: []}])
+    produces("application/json")
+    response(200, "OK", Schema.ref(:UserInfo))
+    response(401, "Unauthorised")
+  end
 
+  def update_game_states(conn, %{"gameStates" => new_game_states}) do
+    user = conn.assigns[:current_user]
+
+    case Cadet.GameStates.update(user, new_game_states) do
+      {:ok, nil} ->
+        text(conn, "OK")
+
+      {:error, {status, message}} ->
+        conn
+        |> put_status(status)
+        |> text(message)
+    end
+  end
+
+  swagger_path :update_game_states do
```

Comment:
Based on how the other Controllers are written, all `swagger_path` documentation should be written below all the controller action methods.